upup/models/cloudup/resources/addons/aws-load-balancer-controller.addons.k8s.io/k8s-1.19.yaml.template

```
@@ -749,8 +749,8 @@ spec:
       containers:
       - args:
         - --cluster-name={{ ClusterName }}
-        - --enable-waf=false
-        - --enable-wafv2=false
+        - --enable-waf={{ WithDefaultBool .AWSLoadBalancerController.EnableWAF false }}
+        - --enable-wafv2={{ WithDefaultBool .AWSLoadBalancerController.EnableWAFv2 false }}
```

Is there a need for `WithDefaultBool` ?
```suggestion
        - --enable-waf={{ .AWSLoadBalancerController.EnableWAF }}
        - --enable-wafv2={{ .AWSLoadBalancerController.EnableWAFv2 }}
```